openmdao/core/group.py

```diff
@@ -689,7 +718,7 @@ def _list_states(self):
         for subsys in self._sorted_sys_iter():
             states.extend(subsys._list_states())
 
-        return sorted(states)
+        return states
```

Comment:
Was this `sorted` originally added for MPI? Just making sure, because those bugs are hard to find.

Reply:
list_states returns only local states so I don't think it'll cause any MPI bugs.  The states are sorted anyway because of _sorted_sys_iter.  There is a separate method called _list_states_allprocs that currently returns a set, which is definitely not sorted, but it's currently only used in SqliteRecorder and it isn't iterated over so that shouldn't be a problem either.